File: app/src/main/java/com/mrk/mahfuj/bangladeshsonbidhan/JsonToJava/Content.java

package com.mrk.mahfuj.bangladeshsonbidhan.JsonToJava;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Content {

    @SerializedName("Content_Type")
    @Expose
    private String contentType;
    @SerializedName("Main_Content")
    @Expose
    private String mainContent;

    public String getContentType() {
        return contentType;
    }

    public void setContentType(String contentType) {
        this.contentType = contentType;
    }

    public String getMainContent() {
        return mainContent;
    }

    public void setMainContent(String mainContent) {
        this.mainContent = mainContent;
    }

}

File: app/src/main/java/com/mrk/mahfuj/bangladeshsonbidhan/Content.kt
package com.mrk.mahfuj.bangladeshsonbidhan

open class Content {
    open var ContentType: String = ""
    open var MainContent: String = ""
    open var Type: Int = 1
    open var TextStyle: String = ""

    constructor(ContentType: String, MainContent: String,Type :Int,TextStyle : String ) {
        this.ContentType = ContentType
        this.MainContent = MainContent
        this.Type =  Type
        this.TextStyle = TextStyle
    }
}